app/src/main/java/nz/org/cacophony/cacophonometer/TestingFragment.java

```diff
@@ -0,0 +1,166 @@
+package nz.org.cacophony.cacophonometer;
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.CompoundButton;
+import android.widget.Switch;
+import android.widget.TextView;
+
+public class TestingFragment extends Fragment {
+
+    private static final String TAG = "TestingFragment";
+
+
+    private Switch swUseTestServer;
+    private Switch swUseVeryFrequentRecordings;
+    private Switch swShortRecordings;
+    private Button btnFinished;
+    private TextView tvMessages;
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        // Inflate the layout for this fragment
+        View view =  inflater.inflate(R.layout.fragment_testing, container, false);
+
+        setUserVisibleHint(false);
+        tvMessages =  view.findViewById(R.id.tvMessages);
+        btnFinished =  view.findViewById(R.id.btnFinished);
+        swUseTestServer =  view.findViewById(R.id.swUseTestServer);
+        swUseVeryFrequentRecordings =   view.findViewById(R.id.swUseVeryFrequentRecordings);
+        swShortRecordings =  view.findViewById(R.id.swShortRecordings);
+
+
+        displayOrHideGUIObjects();
+        btnFinished.setOnClickListener(new View.OnClickListener(){
+            @Override
+            public void onClick(View v) {
+
+                ((AdvancedWizardActivity)getActivity()).nextPageView();
+            }
+        });
+
+
+        swUseTestServer.setOnCheckedChangeListener(new CompoundButton.OnCheckedChangeListener() {
+            @Override
+            public void onCheckedChanged(CompoundButton buttonView, boolean isChecked) {
+                // https://stackoverflow.com/questions/17372750/android-setoncheckedchangelistener-calls-again-when-old-view-comes-back
+                if (!buttonView.isShown()){
+                    return;
+                }
+
+                Util.setUseTestServer(getActivity().getApplicationContext(), isChecked);
+
+//                swShortRecordings.setEnabled(isChecked);
+//                swUseVeryFrequentRecordings.setEnabled(isChecked);
+
+                if (!isChecked){
+                    Prefs prefs = new Prefs(getActivity().getApplicationContext());
+                    prefs.setUseShortRecordings(false);
+//                    swShortRecordings.setChecked(false);
+//                    swShortRecordings.setEnabled(false);
+
+                    prefs.setUseVeryFrequentRecordings(false);
+//                    swUseVeryFrequentRecordings.setChecked(false);
+//                    swUseVeryFrequentRecordings.setEnabled(false);
```

Comment:
Do these commented out sections have to stay around?

Reply:
Removed them and have been through all other code and removed similar